godot/BasicDungeon/02.AndOrganic/AndOrganic.gd

```diff
@@ -17,15 +17,15 @@ func _ready() -> void:
 
 
 func _setup_camera() -> void:
-	camera.position = level.map_to_world(level_size / 2)
-	var z := max(level_size.x, level_size.y) / 8
+	camera.position = level.map_to_local(level_size / 2)
+	var z : float = 8 / max(level_size.x, level_size.y)
 	camera.zoom = Vector2(z, z)
 
 
 func _generate() -> void:
 	level.clear()
 	for vector in _generate_data():
-		level.set_cellv(vector, 0)
+		level.set_cell(0,vector, 0, Vector2i(0,0), 0)
```

Comment:
Keep consistent white spaces and built-in constants: `level.set_cell(0, vector, 0, Vector2i.ZERO, 0)`